packages/integration-tests/api/api.js

```diff
@@ -32,11 +33,19 @@ async function callCumulusApi({ prefix, functionName, payload: userPayload }) {
   process.env.UsersTable = `${prefix}-UsersTable`;
   const userModel = new User();
 
-  const { userName, password } = await userModel.create(fakeUserFactory());
+  const { userName } = await userModel.create(fakeUserFactory());
+
+  process.env.AccessTokensTable = `${prefix}-AccessTokensTable`;
+  const accessTokenModel = new AccessToken();
+
+  const accessTokenRecord = fakeAccessTokenFactory({ username: userName });
```

Comment:
I'm still confused about when we use a username and when we don't., I can set userName to undefined, and the tests pass.

Reply:
Yeah, this is exactly the same case as https://github.com/nasa/cumulus/pull/717#discussion_r237532754. The username doesn't need to be in the access token record but does need to be in the JWT. I can clean up the code to make things clearer